pallets/ajuna-tournament/src/lib.rs

```diff
@@ -592,6 +599,34 @@ pub mod pallet {
 
 			Ok(next_tournament_id)
 		}
+
+		fn try_remove_latest_tournament_for(season_id: &T::SeasonId) -> DispatchResult {
```

Comment:
Is this a simple cleanup extrinsic, so that we can repeatedly call that until we have no more tournaments?

Reply:
Yes that's right, but it will only clean tournaments that have not started